lib/DataBase/schema.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE
+  music_albums (
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    genre VARCHAR(255),
+    label VARCHAR(255),
```

Comment:
In my opinion instead of adding` label` as a `VARCHAR`, you can add `label_id` and make it a foreign key referencing the `label` table because the `label` itself is a class and will have its own table.